source/js/components/fellow-list/fellow-list.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Person from '../../components/people/person.jsx';
+import LoadingIndicator from '../../components/loading-indicator/loading-indicator.jsx';
+
+export default class FellowList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      fellows: null
+    };
+  }
+
+  fetchFellows(query) {
+    Object.assign(query, {
+      'profile_type': `fellow`,
```

Comment:
Shouldn't need single quotes around the property names...

Reply:
I remember linter requires quotes... but I'll try it again without quotes and see if linter flags it

Reply:
k so linter does want us to keep the quotes...